developer/js/source/lexical-model-compiler/build-trie.ts

```diff
@@ -201,7 +204,7 @@ namespace Trie {
    * @param keyFunction Function that converts word forms into indexed search keys
    * @returns A JSON-serialiable object that can be given to the TrieModel constructor.
    */
-  export function buildTrie(wordlist: WordList, keyFunction: Wordform2Key = defaultWordform2Key): object {
+  export function buildTrie(wordlist: WordList, keyFunction: Wordform2Key): object {
```

Comment:
I just realised that this name was a little inconsistent... also needs fixing on line 117 (and elsewhere?)

```suggestion
  export function buildTrie(wordlist: WordList, keyFunction: WordFormToKey): object {
```